client/actions/usersAction.js

```diff
@@ -0,0 +1,164 @@
+import axios from '../utils/index';
+import { getPayload } from '../utils/helpers';
+
+const usersid = getPayload().userId;
+
+// if (getPayload()) {
+//   const userId = getPayload().userId;
+// }
+// else {
+//  return userId = getPayload().userId;
+// }
+
+/**
+ *
+ *
+ * @export getUsers
+ * @returns {Function} returns dispatch
+ */
+export function getUsers() {
+  return (dispatch) => {
+    axios.get('/api/users/')
+      .then((res) => {
+        dispatch({
+          type: 'GET_USERS',
+          payload: res.data.users.rows,
+        });
+      });
+  };
+}
+
+/**
+ *
+ *
+ * @export createUsers
+ * @param {object} data
+ * @returns {Function} returns dispatch
+ */
+export function createUsers(data) {
+  return (dispatch) => {
+    axios.post('/api/users/', data)
+      .then((res) => {
+        dispatch({
+          type: 'CREATE_USERS',
+          payload: res.data.newUser
+        });
+      });
+  };
+}
+
+/**
+ *
+ *
+ * @export
```

Comment:
The JS Doc tags could be more descriptive to explain exactly what the method does, refer [here](http://usejsdoc.org/about-block-inline-tags.html)